pkg/vip/ddns.go

```diff
@@ -41,14 +42,9 @@ func (ddns *ddnsManager) Start() (string, error) {
 	// channel to wait for IP
 	ipCh := make(chan string)
 
-	client := DHCPClient{
-		// send host name, not current os.Hostname, but configured name from user
-		Hostname:  ddns.network.DDNSHostName(),
-		Interface: iface,
-		OnBound: func(lease *Lease) {
-			ipCh <- lease.ClientIP
-		},
-	}
+	client := NewDHCPClient(iface, false, "", func(lease *nclient4.Lease) {
+		ipCh <- lease.ACK.YourIPAddr.String()
+	})
```

Comment:
I do not familiar with ddns and not sure how to use a DHCP client in ddns. @thebsdbox

Reply:
This looks OK to me.